jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/spec/resource/requestmatching/JAXRSClientIT.java

```diff
@@ -429,7 +429,7 @@ public void optionsOnSubResourceTest() throws Fault {
     invoke();
     boolean foundGet = false;
     for (String header : getResponseHeaders())
-      if (header.startsWith(HttpHeaders.ALLOW))
+      if (header.toLowerCase().startsWith(HttpHeaders.ALLOW.toLowerCase()))
```

Comment:
In this case it's likely not critical, but we might want to use `toLowerCase(Locale.ROOT)`.